File: app/src/main/java/com/example/schedule/ui/SplashActivity.java
package com.example.schedule.ui;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

import com.example.schedule.SQL.SQLManager;

import androidx.appcompat.app.AppCompatActivity;

public class SplashActivity extends AppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        if (getSharedPreferences(SQLManager.SHARED_PREFERENCES_TABLES, Context.MODE_PRIVATE).getAll().isEmpty()) {
            startActivity(new Intent(this, LoginActivity.class));
        } else {
            startActivity(new Intent(this, MainActivity.class));
        }
        finish();

    }
}

File: app/src/main/java/com/example/schedule/ui/SplashActivity.kt
package com.example.schedule.ui

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.schedule.SQL.SQLManager
import com.example.schedule.ui.login.LoginActivity

class SplashActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (getSharedPreferences(SQLManager.SHARED_PREF_DB_TABLE_NAME, MODE_PRIVATE).all.isEmpty()) {
            startActivity(Intent(this, LoginActivity::class.java))
        } else {
            startActivity(Intent(this, MainActivity::class.java))
        }
        finish()
    }
}
